### PR TITLE
dl00's First Goodie

### PR DESCRIPTION
**Pull request by dl00**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**

It does as instructed by the Goodie Quickstart tutorial (although I made some small modifications to the return text)

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It inspires a potential developer to make more goodies

**What is the data source for your Instant Answer? (Provide a link if possible)**

String supplied in the Goodie

**Why did you choose this data source?**

Because it is constant

**Are there any other alternative (better) data sources?**

No

**What are some example queries that trigger this Instant Answer?**

duckduckhack dl00

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Me and my friends :)

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

No

**Which existing Instant Answers will this one supercede/overlap with?**

It is almost exactly the same as all the other IsAwesome templates

**Are you having any problems? Do you need our help with anything?**

No

**Where did you hear about DuckDuckHack? (For first time contributors)**

Stumbled upon it when examining duckduckgo in-depth

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
duckduckhack dl00 at duckduckgo

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Linux
        [X] Firefox
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/dl00.pm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+primary_example_queries "duckduckhack dl00";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/dl00.pm";
+attribution github => ["https://github.com/dl00", "dl00"];
+
+triggers start => "duckduckhack dl00";
+
+handle remainder => sub {
+	return if $_;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/dl00.pm]: Can you switch these hard-tabs out for 4 spaces?

**Comment by mintsoft** [comment]
@dl00 Congrats on your first goodie! Can you just switch the hard-tabs to soft-tabs then we'll get it merged & live!

**Comment by dl00** [comment]
Should have fixed them (if I understand how github pull requests work)

By the way, the reason the hard tabs were in there was because I was having issues with the editor, the new files were not being recognized by codio... I probobly should have listed that is a problem.

**Comment by mintsoft** [comment]
@moollaza any idea here about the Codio issue?

**Comment by moollaza** [comment]
@mintsoft I'm not sure right now, there may have been Codio updates while I was away so I'll play around with Codio to see if anying funny happens. I haven't seen any similar complaints yet though so it may have been a 1-time thing.

>the new files were not being recognized by codio...

@dl00 could you expand on that -- what exactly do you mean? Were they not visible in the file tree?

**Comment by dl00** [comment]
So, on the tutorial, I ran the new DuckPAN Goodie script just like it said. Then it told me to open the file with Ctrl-O. I typed in my github username but it would not find the files. Then I went into the file tree, and the files were not there either. I eventually had to navigate there in the terminal and edit them with Vim.

The problem seems to have gone away, however. Just thought I should mention though. Sorry for being so vague before.

**Comment by moollaza** [comment]
>Sorry for being so vague before.

Ah it's no problem at all. Thanks a lot for explaining what happened. That's certainly some odd behaviour.

I just tried generating a Goodie using your username and it worked so I guess everything is OK for now!

**Comment by dl00** [comment]
Well, I did just figure out how to fix it if it happens again:

Project -> Resync File Tree

**Comment by moollaza** [comment]
@dl00 congrats, it's live! https://duckduckgo.com/?q=duckduckhack+dl00&t=ffcm&ia=answer

**Pull request merged**